src/components/Login/Login.js

```diff
@@ -1,26 +1,66 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
+import fetch from 'isomorphic-fetch';
 import styles from  './Login.css';
 import useStoreon from 'storeon/react'
 import Header from '../common/Header';
 
 const Login = () => {
     const { user, dispatch } = useStoreon('user');
+    const usernameEl = useRef(null);
```

Comment:
Using controlled components (state) here will be good to validate the username and password field on each key strokes.
I feel using unControlled components, validation can be tricky.

Reply:
Im trying to avoid the need of class instances and use simple sfc since the use case if also very simple.

We wont need any complex validations here because its not a form. But just a login field.